container/go/cmd/puller/puller.go

```diff
@@ -0,0 +1,57 @@
+package puller
+
+import (
+	"fmt"
+	"log"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1/cache"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
+	"github.com/google/go-containerregistry/pkg/v1/tarball"
+)
+
+const iWasADigestTag = "i-was-a-digest"
```

Comment:
Also copy the comment above it.

Reply:
Also make a note that this file is mostly copied from https://github.com/google/go-containerregistry/blob/master/pkg/crane/pull.go